src/coffee/bootstrap-tour.coffee

```diff
@@ -475,7 +477,8 @@
 
       $element
       .popover(
-        placement: step.placement
+        placement: ->
+          return if step.smartPlacement is true then "auto #{step.placement}" else step.placement
```

Comment:
You're right, I should check if auto is already set.
I think that the function is needed since the smartPlacement option could be set differently for each step.